Lib/Rabbit4000/Crypto/SHA2.LIB

```diff
@@ -145,14 +145,16 @@ void sha256_add(sha256_context __far *ctx, const uint8_t __far *input,
 #define F1_S3(x,y,z) _sha256_F1_S3(x,y,z)
 uint32_t _sha256_S0(uint32_t x);
 uint32_t _sha256_S1(uint32_t x);
-
-_sha2_debug
+root _sha2_debug
+// result in jkhl and not bcde
+// all arguments in registers and no stack use

```

Comment:
Since `_sha256_F0_S2()` is no longer callable from C (due to not using Dynamic C's ABI for function calls), you should remove the C prototypes and have the function entirely contained in a `#asm` block, using `_sha256_F0_S2::` as the opening label (ala `_sha256_S0`).

Be sure to document the interface to the function (e.g., pass parameters x, y, and z in registers PX, PY, and PZ; return value in JKHL) so it's easy to validate the implementation and calling code.